CRM/Utils/Check/Component/Schema.php

```diff
@@ -145,34 +145,28 @@ public function checkSmartGroupCustomFieldCriteria() {
             $customField = civicrm_api3('CustomField', 'getsingle', [
               'sequential' => 1,
               'id' => $field['cfid'],
-            ]);
-            $fieldName = ts('<a href="%1" title="Edit Custom Field"> %2 </a>', [
-              1 => CRM_Utils_System::url('civicrm/admin/custom/group/field/update',
-                "action=update&reset=1&gid={$customField['custom_group_id']}&id={$field['cfid']}", TRUE
-              ),
-              2 => $customField['label'],
-            ]);
+           ]);
+            $url = CRM_Utils_System::url('civicrm/admin/custom/group/field/update', "action=update&reset=1&gid={$customField['custom_group_id']}&id={$field['cfid']}", TRUE);
+            $fieldName = '<a href="' . $url . '" title="' . ts('Edit Custom Field', ['escape' => 'js']) . '">' . $customField['label'] . '</a>';
           }
           catch (CiviCRM_API3_Exception $e) {
-            $fieldName = ' <span style="color:red"> - Deleted - </span> ';
+            $fieldName = '<span style="color:red">' . ts('Deleted') . ' - ' . ts('Field ID %1', [1 => $field['cfid']]) . '</span> ';
           }
         }
-        $groupEdit = '<a href="' . CRM_Utils_System::url('civicrm/contact/search/advanced', "?reset=1&ssID={$field['ssid']}", TRUE) . '" title="' . ts('Edit search criteria') . '"> <i class="crm-i fa-pencil" aria-hidden="true"></i> </a>';
-        $groupConfig = '<a href="' . CRM_Utils_System::url('civicrm/group', "?reset=1&action=update&id={$id}", TRUE) . '" title="' . ts('Group settings') . '"> <i class="crm-i fa-gear" aria-hidden="true"></i> </a>';
+        $groupEdit = '<a href="' . CRM_Utils_System::url('civicrm/contact/search/advanced', "?reset=1&ssID={$field['ssid']}", TRUE) . '" title="' . ts('Edit search criteria', ['escape' => 'js']) . '"> <i class="crm-i fa-pencil" aria-hidden="true"></i> </a>';
+        $groupConfig = '<a href="' . CRM_Utils_System::url('civicrm/group', "?reset=1&action=update&id={$id}", TRUE) . '" title="' . ts('Group settings', ['escape' => 'js']) . '"> <i class="crm-i fa-gear" aria-hidden="true"></i> </a>';
```

Comment:
It was there before but since editing anyway can you remove the extra '?' in the url params that's before the reset=1 ?